### PR TITLE
Get all debug messages

### LINKED ISSUE #733: Ltac printing - where is it displayed?
Hi,

First, thank you for all the hard work on the extension!

I recently updated to modern VsCoq (vscoq-language-server 2.1.0, coq 8.18, VsCoq 2.1), and I had trouble to figure out where the Ltac debugging messages are showing, typically in the following snippet:

```Coq
  Goal True.
     idtac "Printing something".
```

In previous versions, there used to be a buffer for those Ltac debugging outputs, but I can't seem to find it anymore. Is it a deprecated feature, or maybe there is a more modern way to achieve a similar effect, or I just missed the place where the output would be displayed?

### PR DESCRIPTION
**Pull request by rtetley**
Closes #733 #819 #820 #815.

### CODE REVIEW

File: language-server/dm/log.mli [hunk 1]
```diff
@@ -26,6 +26,8 @@ val handle_event : event -> unit
 val worker_initialization_begins : unit -> unit
 val worker_initialization_done : fwd_event:(event -> unit) -> unit
 
+val debug : event Sel.Event.t
```
Comment (gares) [inline language-server/dm/log.mli:29]:
```suggestion
(* debug messages coming from either the language server of Coq *)
val debug : event Sel.Event.t
```

**Comment by TDiazT** [comment]
Not sure if intended, but debug logs are printed only once apparently.
For instance, in the example of #815 :
```coq
Class A.
Instance a : A := {}.

Set Typeclasses Debug.
Goal A. apply _. Qed.
```
The first time `apply _` is interpreted, it shows the log, but subsequent executions do not. 
Not sure if this a Coq or a vscoq thing.

**Comment by TDiazT** [comment]
> Not sure if intended, but debug logs are printed only once apparently. For instance, in the example of #815 :
> 
> ```coq
> Class A.
> Instance a : A := {}.
> 
> Set Typeclasses Debug.
> Goal A. apply _. Qed.
> ```
> 
> The first time `apply _` is interpreted, it shows the log, but subsequent executions do not. Not sure if this a Coq or a vscoq thing.



> Not sure if intended, but debug logs are printed only once apparently. For instance, in the example of #815 :
> 
> ```coq
> Class A.
> Instance a : A := {}.
> 
> Set Typeclasses Debug.
> Goal A. apply _. Qed.
> ```
> 
> The first time `apply _` is interpreted, it shows the log, but subsequent executions do not. Not sure if this a Coq or a vscoq thing.

Actually, nvm, I think it's smart enough that it only shows on code change haha. I was trying it out without any code change but then it works when one does change the file.

**Comment by rtetley** [comment]
I'll merge and will find a way to get the Document URI later

**Pull request merged**